experimenter/experimenter/theme/static_src/src/index.js

```diff
@@ -1,19 +1,94 @@
 import Alpine from "alpinejs";
 
-Alpine.data("dropdown", () => ({
+Alpine.data("sidebar", () => ({
   open: false,
 
   toggle() {
     this.open = !this.open;
   },
 }));
 
+Alpine.data("modal", () => ({
+  isModalOpen: false,
+  trapCleanup: null,
+  oldData: null,
+  newData: null,
+  
+  openModal() {
+    this.isModalOpen = true
+    this.trapCleanup = focusTrap(document.querySelector('#modal'))
+  },
+  closeModal() {
+    this.isModalOpen = false
+    this.oldData = null;
+    this.newData = null;
+    this.trapCleanup()
+  },
+}));
+
 Alpine.data("redirect", () => ({
   redirectToReactRoute(slug, pageSlug) {
     window.location.href = `/nimbus/${slug}/edit/${pageSlug}`;
   },
 }));
 
+Alpine.data('slider', () => ({
+                
+  // set initial tab
+  tab: 1,
+  
+  // slider tabs
+  tabs: [...document.querySelectorAll('nav[role=tablist] a[role=tab]')],
+  
+  init() {
+      // initialize main function
+      this.changeSlide()
+  },
+  
+  // main function
+  changeSlide() {
```

Comment:
Why is this separated into a different function?

Reply:
This was also from an example, I was trying different things for the modal so I just kept it as it is. I'll update this as well